addons/web/views/report_templates.xml

```diff
@@ -192,7 +192,7 @@
                     <li t-if="company.phone" class="list-inline-item"><i class="fa fa-phone" role="img" aria-label="Phone" title="Phone"/> <span t-field="company.phone"/></li>
                     <li t-if="company.email" class="list-inline-item"><i class="fa fa-at" role="img" aria-label="Email" title="Email"/> <span t-field="company.email"/></li>
                     <li t-if="company.website" class="list-inline-item"><i class="fa fa-globe" role="img" aria-label="Website" title="Website"/> <span t-field="company.website"/></li>
-                    <li t-if="company.vat" class="list-inline-item"><i class="fa fa-building-o" role="img" aria-label="Fiscal number"/><t t-esc="company.country_id.vat_label or 'Tax ID'"/>: <span t-field="company.vat"/></li>
+                    <li t-if="company.vat" class="list-inline-item"><i class="fa fa-building-o" role="img" aria-label="Fiscal number"/><t t-esc="company.country_id.vat_label or 'Tax&#160;ID'"/>: <span t-field="company.vat.replace(' ','&#160;')"/></li>
```

Comment:
I'd go for:
```
<t t-esc="(company.country_id.vat_label or 'Tax ID').replace(' ','&#160;')"/>
```

Reply:
Using &nbsp; will give an error "lxml.etree.XMLSyntaxError: Entity 'nbsp' not defined", because it is an html entity, using &#160; which is the corresponding xml one or \N{NO-BREAK SPACE} is fine.
Updated